scripts/generator/templates/template-dev-bundle-pom.xml

```diff
@@ -279,6 +279,7 @@
                 <configuration>
                     <excludes>
                         <exclude>**/*Fake*</exclude>
```

Comment:
Can this "Fake" be removed now? I think it was for the same purpose when the class was named differently

Reply:
Oh, there is also a FakeAppConf. Why aren't the annotations on that class?

Reply:
Done